### DISCUSSION TITLE
blog/2023/230428-mokabot-arcaea/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#8] [Announcements]
# blog/2023/230428-mokabot-arcaea/

mokabot 自诞生之初就已经实装了 Arcaea 查分功能，直到 2023 年 4 月末，伴随着 ArcaeaUnlimitedAPI 的关闭而彻底终结，持续了近 3 年的 Arcaea 查分功能最终划上了句号。

https://blog.arisa.moe/blog/2023/230428-mokabot-arcaea/

**Comment by InariAimu** [comment]
No one cares? I do.
